issue_06/script.js

```diff
@@ -11,21 +11,29 @@ const contents = [{
 }];
 
 const ul = document.getElementById('js-ul');
-const checkContents = new Promise((resolve) => {
-    resolve(contents);
-});
-checkContents.then((value) => {
+const promiseSettings = () => {
+    return new Promise((resolve) => {
```

Comment:
```suggestion
const promiseSettings = () => {
const contents = [{
    to: "bookmark.html",
    img: "1.png",
    alt: "画像1",
    text: "ブックマーク"
}, {
    to: "message.html",
    img: "2.png",
    alt: "画像2",
    text: "メッセージ"
}];
    return new Promise((resolve) => {
```
nits
This will make it easier to read.

Reply:
I found this one easier to read, so I fixed it.
https://github.com/yoshikawa-akr/js-lessons/pull/6/commits/92155ae3c404d109745778dc5f2f2c653f615390